src/critics/prefer_forward_critic.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 FastSense, Samsung Research
+
+#include "mppic/critics/prefer_forward_critic.hpp"
+
+namespace mppi::critics
+{
+
+void PreferForwardCritic::initialize()
+{
+  auto node = parent_.lock();
+
+  auto getParam = utils::getParamGetter(node, name_);
+  getParam(power_, "prefer_forward_cost_power", 1);
+  getParam(weight_, "prefer_forward_cost_weight", 20.0);
```

Comment:
Is this a reasonable penalty? I thought it worked pretty OK as is, so I would have thought a small value here would go a long way

Reply:
For me it works better with this value, but surely it would requires more tests. I think to start experimenting with defaults while all critics are done